knpuniversity/oauth2-client-bundle/1.20/manifest.json

```diff
@@ -0,0 +1,14 @@
+{
+	"bundles": {
+			"KnpU\\OAuth2ClientBundle\\KnpUOAuth2ClientBundle": ["all"]
+	},
+	"copy-from-recipe": {
+			"config/": "%CONFIG_DIR%/"
+	},
+	"env": {
+			"#1": "Register for a facebook oauth app:",
+			"#2": "https://developers.facebook.com/apps/",
+			"FB_ID": "facebook-app-id",
+			"FB_SECRET": "facebook-secret"
+	}
```

Comment:
Because we don't know what client they'll use, I think we shouldn't make any changes to the `.env` file. Instead, we should update the README to talk about this.